### ISSUE TITLE
console is visible when js is disabled

### ISSUE THREAD

**Issue by daveriedstra** [ueokande/vim-vixen#504]
### I'm opening this issue because:

- [x] I'll report a bug
- [ ] I'll propose a new feature

### Description
When Javascript is disabled in FF, the vimvixen console iframe is visible as a white box in the top left corner of the page

### Failure Information (for bugs)
n/a

#### Steps to Reproduce

1. go to some page
2. disable javascript via dev tools (reloads page)
3. observe white box in top left corner of page

#### System configuration

- Operating system: mac os high sierra
- Firefox version: 63.0.1 (64 bit)
- Vim-Vixen version: 0.19

#### Console logs
```
Loading failed for the <script> with source “moz-extension://7ee66385-1ef4-7740-ad80-0a5bf1fca1d6/build/console.js”.
```

**Comment by shahab-vahedi** [comment]
Same happens here. However, for me it is the bottom left of the page and there is no Javascript disabled.
For instance, it happens on [this page](https://dustri.org/b/defeating-lincrackme3-with-radare2.html).

**Console log**
```
Content Security Policy: Couldn’t process unknown directive ‘trusted-types’
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“default-src”).
GET https://dustri.org/b/theme/css/skeleton.css [HTTP/2.0 200 OK 0ms]

GET https://dustri.org/b/theme/css/style.css  [HTTP/2.0 200 OK 0ms]

Stylesheets without CSSOM changes reparsed to check for errors. Refresh the page to also see errors from stylesheets changed from CSSOM and from style attributes.
GET https://dustri.org/b/theme/fonts/raleway.woff2  [HTTP/2.0 200 OK 0ms]
```

**Comment by ueokande** [comment]
Please enable JavaScript to use vim-vixen.

**Comment by daveriedstra** [comment]
> Please enable JavaScript to use vim-vixen.

Naturally js must be enabled to use the extension, but my suggestion is that the extension should not have an effect on tabs where js is disabled.

**Comment by ueokande** [comment]
Got it, it may be fixed with #645.

**Comment by ueokande** [comment]
This issue is already fixed on #571
